packages/card-support/src/constants.ts

```diff
@@ -1,7 +1,7 @@
 // OPENSRP API endpoints
-export const OPENSRP_URL_USER_ASSIGNMENT = '/organization/user-assignment';
-export const OPENSRP_URL_CLIENT_SEARCH = '/client/search';
-export const OPENSRP_URL_LOCATION_HIERARCHY = '/location/hierarchy';
+export const OPENSRP_URL_USER_ASSIGNMENT = 'organization/user-assignment';
+export const OPENSRP_URL_CLIENT_SEARCH = 'client/search';
+export const OPENSRP_URL_LOCATION_HIERARCHY = 'location/hierarchy';
```

Comment:
Do any tests exist to show that URLS containing these strings are well formed?

Reply:
the base url set in the env file has a trailing foward slash, i would have used the base url set in env but since this is at package level we are unable to test that

Reply:
Okay.  Sounds like we need a way to test such things in the long term.  Something to think about for the future.

Reply:
as part of the future thoughts, I think we could use (node's/other OSS) url parsing utils to do things like building urls